diff --git a/scripts/vars.mjs b/scripts/vars.mjs
index 1111111..2222222 100644
--- a/scripts/vars.mjs
+++ b/scripts/vars.mjs
@@ -10,13 +10,10 @@ const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
 
 const electronInfo = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../node_modules/electron/package.json')))
 
-export let version = childProcess.execSync('git describe --tags', { encoding:'utf-8' })
-version = version.substring(1).trim()
-version = version.replace('-', '-c')
-
-if (version.includes('-c')) {
-    version = semver.inc(version, 'prepatch').replace('-0', `-nightly.${process.env.REV ?? 0}`)
-}
+export let version = process.env.TABBY_PACKAGE_VERSION
+if (!version) {
+    throw new Error('TABBY_PACKAGE_VERSION must be set')
+}
 
 export const builtinPlugins = [
     'tabby-core',
